cnames_active.js

```diff
@@ -2634,6 +2634,7 @@ var cnames_active = {
   "s3swa": "s3swa-ict-upgifter.github.io/snow-eater",
   "s4swa": "s4swa.github.io",
   "saaskit": "cname.vercel-dns.com", // noCF
+  "sacgencode":"sacgencode.github.io",
```

Comment:
```suggestion
  "sacgencode": "sacgencode.github.io",
```
As the CI error notes, you're missing a space here